src/library/trackcollection.h

```diff
@@ -99,6 +100,7 @@ class TrackCollection : public QObject {
 #endif // __SQLITE3__
 
   private:
+    const static unsigned short int MASK = 0xF000;
```

Comment:
This can be local as well. 
Is there a definition you can use instead? 
If not, you may consider to combine your value by |-ing the single bits.